x-pack/plugins/observability/public/hooks/slo/use_fetch_active_alerts.ts

```diff
@@ -75,37 +72,32 @@ export function useFetchActiveAlerts({ sloIds = [] }: Params): UseFetchActiveAle
                     },
                   },
                 ],
-                should: [
-                  {
-                    terms: {
-                      'kibana.alert.rule.parameters.sloId': sloIds,
-                    },
+                should: sloIdsAndInstanceIds.map(([sloId, instanceId]) => ({
+                  bool: {
+                    filter: [
+                      { term: { 'slo.id': sloId } },
+                      { term: { 'slo.instanceId': instanceId } },
```

Comment:
Don't we need to prefix with `kibana.alert.rule.parameters` ?

Reply:
Nope... I added these fields to the AAD index: https://github.com/elastic/kibana/pull/163434/files#diff-617cdbe812afb1f1786df0d507bd9fa6e8e065098e98206141cccaf7835c62e8